mod/randprof.php

```diff
@@ -13,8 +13,8 @@ function randprof_init(App $a)
 	$x = GContact::getRandomUrl();
 
 	if ($x) {
-		goaway(Contact::magicLink($x));
+		$a->internalRedirect(Contact::magicLink($x));
```

Comment:
There are several calls inside `Contact::magicLink()`. I'm not sure if this is REALLY a relative path.

After some research I have the fear that it depends and can be both ;-)

Reply:
Same suggestion use a filter and add a `@TODO` to refactor the behavior later.